rl-institut/oemof-B3
# Implement github actions

Fix #6 

Changes proposed in this PR:
- Add github actions workflow including: 
     - run tests with pytest
     - add test coverage
     - check linting with black
- Remove `test_requirements.txt` and add `extras_require` to `setup.py`

The checks are done for python 3.7 and 3.8 currently, this can be adapted of course.
